pkg/apis/sessions/interfaces.go

```diff
@@ -0,0 +1,12 @@
+package sessions
+
+import (
+	"net/http"
+)
+
+// SessionStore is an interface to storing user sessions in the proxy
+type SessionStore interface {
```

Comment:
Similar to the Cookie comment, does each method need to have the `Session` suffix as it's also a bit of unneeded duplication imo.

Reply:
Totally happy to rename these, these were just the first names that came off the top of my head

Reply:
Have renamed!